### PR TITLE
✨ add notes history for Promise.withResolvers()

### PR DESCRIPTION
**Pull request by ctcpip**

### CODE REVIEW

File: finished-proposals.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Finished proposals are proposals that have reached stage 4, and are included in
...
 | [Array Grouping][array-grouping]                                         | Justin Ridgewell                                       | Justin Ridgewell<br />Jordan Harband                    | [November 2023][array-grouping-notes]       | 2024                      |
-| [`Promise.withResolvers`][promise-defer]                                 | Peter Klecha                                           | Peter Klecha                                            | [November 2023][promise-defer-notes]        | 2024                      |
+| [`Promise.withResolvers`][promise-defer]                                 | Peter Klecha                                           | Peter Klecha                                            | [November 2023][promise-defer-notes-2023-11]<br>[July 2023][promise-defer-notes-2023-07]<br>[May 2023][promise-defer-notes-2023-05]<br>[March 2023][promise-defer-notes-2023-03]       | 2024                      |
```
Comment (ljharb) [inline finished-proposals.md:62]:
```suggestion
| [`Promise.withResolvers`][promise-defer]                                 | Peter Klecha                                           | Peter Klecha                                            | <sub><ul><li>[November 2023][promise-defer-notes-2023-11]</li><li>[July 2023][promise-defer-notes-2023-07]</li><li>[May 2023][promise-defer-notes-2023-05]</li><li>[March 2023][promise-defer-notes-2023-03]</li></ul></sub>       | 2024                      |
```
maybe?

Comment (ctcpip) [reply finished-proposals.md:62]: overlooking the fact that that is invalid HTML, the rendering leaves something to be desired...

Screenshot 2024-02-14 at 2 26 44 PM

Comment (ljharb) [reply finished-proposals.md:62]: oof, yeah. maybe putting it inside a `<details>`?

Comment (ctcpip) [reply finished-proposals.md:62]: not possible

**Comment by ljharb** [comment]
Looks pretty good! altho at this point it seems better to use the `<br />`s than the commas - on my screen it's always one per line anyways.

**Comment by ctcpip** [comment]
Screenshot 2024-02-14 at 5 20 49 PM

**Comment by ljharb** [comment]
very strange, keep the commas then.

File: finished-proposals.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Finished proposals are proposals that have reached stage 4, and are included in
...
 | [Resizable and growable ArrayBuffers][resizable]                         | Shu-yu Guo                                             | Shu-yu Guo                                              | [September 2023][resizable-notes]           | 2024                      |
 | [Array Grouping][array-grouping]                                         | Justin Ridgewell                                       | Justin Ridgewell<br />Jordan Harband                    | [November 2023][array-grouping-notes]       | 2024                      |
-| [`Promise.withResolvers`][promise-defer]                                 | Peter Klecha                                           | Peter Klecha                                            | [November 2023][promise-defer-notes]        | 2024                      |
+| [`Promise.withResolvers`][promise-defer]                                 | Peter Klecha                                           | Peter Klecha                                            | <sub>[2023-11][promise-defer-notes-2023-11], [2023-07][promise-defer-notes-2023-07], [2023-05][promise-defer-notes-2023-05], [2023-03][promise-defer-notes-2023-03]       | 2024</sub>                     |
```
Comment (ljharb) [inline finished-proposals.md:62]:
```suggestion
| [`Promise.withResolvers`][promise-defer]                                 | Peter Klecha                                           | Peter Klecha                                            | <sub>[2023-11][promise-defer-notes-2023-11], [2023-07][promise-defer-notes-2023-07], [2023-05][promise-defer-notes-2023-05], [2023-03][promise-defer-notes-2023-03]</sub>       | 2024                     |
```

**Review by christceder — APPROVED** [review]

**Review by ljharb — APPROVED** [review]
k i tweaked it a bit and i'm happy with it now; i'm going to land this, and would love further PRs to flesh out the notes history for all the proposals (in all the tables)

**Review by KurlyKyleX — COMMENTED** [review]
True

**Review by KurlyKyleX — COMMENTED** [review]
True

**Pull request merged**